### PR TITLE
[rllib] Add three new long-running stress tests {APEX, IMPALA, PBT}

### PR DESCRIPTION
**Pull request by ericl**
- workload_apex.py -- tests APEX with image observations
- workload_impala.py -- tests IMPALA with remote envs
- workload_pbt.py -- PBT actor workload

These are all running on a single raylet for now. If they prove to be very stable over many days, we can try bumping them up to multiple emulated raylets.

I'm currently running these now to see if we run into any issues. Seems fine up to 24 hours.

### CODE REVIEW

File: doc/source/rllib-examples.rst [hunk 1]
```diff
@@ -65,5 +65,7 @@ Community Examples
    Example of training robotic control policies in SageMaker with RLlib.
 - `StarCraft2 <https://github.com/oxwhirl/smac>`__:
    Example of training in StarCraft2 maps with RLlib / multi-agent.
+- `NeuroCuts <https://github.com/xinjin/neurocuts-code>`__:
```
Comment (ericl) [inline doc/source/rllib-examples.rst]: unrelated doc update

File: ci/long_running_tests/check_workloads.sh [hunk 2]
```diff
@@ -4,21 +4,33 @@
 
 ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 
-pushd "$ROOT_DIR"
+if [ "$1" == "--load" ]; then
+    check_load=true
```
Comment (ericl) [inline ci/long_running_tests/check_workloads.sh]: This gives a summary like
```
workload1:  02:23:40 up 53 min,  2 users,  load average: 0.00, 0.02, 0.84
workload2:  02:23:41 up 53 min,  2 users,  load average: 0.00, 0.00, 0.07
workload3:  02:23:43 up 53 min,  2 users,  load average: 0.00, 0.00, 0.22
workload_apex:  02:23:45 up 53 min,  3 users,  load average: 16.85, 7.67, 3.54
workload_impala:  02:23:46 up 53 min,  2 users,  load average: 13.23, 12.72, 11.49
workload_pbt:  02:23:48 up 53 min,  2 users,  load average: 11.04, 11.63, 10.06
```

Which makes it easy to check if everything ok by inspecting if the most recent load is >>1.

File: python/ray/scripts/scripts.py [hunk 3]
```diff
@@ -685,7 +685,7 @@ def stack():
 # Call sudo to prompt for password before anything has been printed.
 sudo true
 workers=$(
-    ps aux | grep ' ray_' | grep -v grep
+    ps aux | grep -E ' ray_|default_worker.py' | grep -v grep
```
Comment (ericl) [inline python/ray/scripts/scripts.py]: Make sure to stack the un-named workers too.

**Review by robertnishihara — APPROVED** [review]
This looks great!

**Comment by robertnishihara** [comment]
I tried it out and it works nicely.

**Comment by ericl** [comment]
For the record, after a week:
- IMPALA is still running, at 1.6 billiion time steps
- APEX is still running, at 300 million time steps
- PBT crashed with out of disk space (oops, we should fix that)

**Pull request merged**